Blog/view/logIn.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+session_start();
+
+//include('Blog/include/functions.php');
+include('Blog/include/loginFunctions.php');
+					$Errors = array();
+
+					if (isset($_REQUEST['logInSubmit'])) {
+						if ($_REQUEST['userName'] == ''){
+							$Errors['userName'] = "required";
+							echo "Username Required <br/>";
```

Comment:
Normally you wouldn't echo this stuff here, you'd just save it to the `$Errors` array. Then you can echo it down with the form. Right now you're echoing it before you've even echoed <html>